.github/stale.yml

```diff
@@ -0,0 +1,18 @@
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 7
+# Number of days of inactivity before a stale Issue or Pull Request is closed
+daysUntilClose: 7
+# Only issues or pull requests with all of these labels are checked if stale. Defaults to `[]` (disabled)
+onlyLabels:
+ - stat:awaiting response
```

Comment:
did you mean this one? https://github.com/bazelbuild/bazel/labels/awaiting-user-response

Reply:
Done